lib/user_agent/browsers/webkit.rb

```diff
@@ -13,6 +13,8 @@ def browser
           'Android'
         elsif platform == 'BlackBerry'
           platform
+        elsif firefoxOniOS?
+          'Firefox'
```

Comment:
I think this should be `FxiOS` or `Firefox for iOS` as it is a distinct browser from Firefox